system/datatypes/peer_reviewed_article.php

```diff
@@ -1158,14 +1158,25 @@ private function createDiscussionCommentForm(DocPage $discussion, array $data =
 	 */
 	private function getCommentFormHtml($type, DocPage $discussion = null) {
 		$comment = __('art-comment');
+		$infoCommentRules = makeInfoButton('help-comment-rules');
 		$new = $discussion === null;
 		$commentFieldName = self::FIELD_NAME_DISCUSSION_COMMENT . '/' . ($new ? 'new_' . $type : $discussion->getId());
+		if (!isUser()) {
+		$html = <<<EOF
+			<div class="new-comment $type">
+			<div>
+				<strong><label for="$commentFieldName"> $comment </label></strong>$infoCommentRules<br/><textarea name="$commentFieldName" id="$commentFieldName" cols="36" rows="4"></textarea>
+			</div>
+EOF;
+		}
+		if (isUser()) {
```

Comment:
I think this if block should be smaller, so there is less duplication of (HTML) code. Probably setting `$infoCommentRule` to the empty string for logged in users is sufficient here.